src/cdi_ndi/NDI_TN.cc

```diff
@@ -152,8 +180,9 @@ NDI_TN::NDI_TN(const std::string &gendir_in, const std::string &library_in,
   q_reaction *= 1000.; // MeV -> keV
 
   //! Specify multigroup option
-  ndi_error = NDI2_set_option(dataset_handle, NDI_COLLAPSE,
-                              mg_form_map[mg_form].c_str());
+  ndi_error = NDI2_set_float64_vec_option(dataset_handle, NDI_COLLAPSE,
```

Comment:
Random question: Why do all these NDI calls have a "2" in them? :D

Reply:
I remember this came up on Ancient Aliens recently -- some elaborate numerological argument that I forget -- but I prefer to just think of it as referring to NDI currently being in version 2

Reply:
THAT'S WHAT I WAS AFRAID OF (the aliens, not version 2 of NDI)